### PR TITLE
Sagemaker Image example

### PR DESCRIPTION
**Pull request by riteshkarval**
Readme fixes

### CODE REVIEW

**Comment by riteshkarval** [comment]
1. Sagemaker Image example
2. Alerts for image data type
3. Made Tensorflow image for IDE to run all the examples in one NB.

File: insurance_datasources/README.md [hunk 1]
```diff
@@ -131,7 +131,7 @@
 DKube provides Python SDK for creating a modelmonitor programmatically. You could also choose to create a modelmonitor from the DKube UI. Follow one of the following workflows as per your need.
 
 - **Create Modelmonitor using SDK**
-1. From **workspace/insurance/insurance** open [modelmonitor.ipynb](https://github.com/oneconvergence/dkube-examples/tree/monitoring/insurance_datasources/modelmonitor.ipynb) and run all the cells. New model monitor will be created.
+1. From **workspace/monitoring-examples/insurance** open [modelmonitor.ipynb](https://github.com/oneconvergence/dkube-examples/tree/monitoring/insurance_datasources/modelmonitor.ipynb) and run all the cells. New model monitor will be created.
```
Comment (songole) [inline insurance_datasources/README.md]: you mean insurance_datasources?

File: insurance_datasources/README.md [hunk 2]
```diff
@@ -146,7 +146,7 @@ DKube provides Python SDK for creating a modelmonitor programmatically. You coul
    - New model version will be created for 'insurance-model' model
    - Follow step 4 if you want to retrain via sdk or step 5 if you want to retrain using UI.
 4. **SDK**:
-   - From **workspace/insurance/insurance** open modelmonitor.ipynb and run the Retraining cell. It will update the dataset and model version in the existing model monitor.
+   - From **workspace/monitoring-examples/insurance** open modelmonitor.ipynb and run the Retraining cell. It will update the dataset and model version in the existing model monitor.
```
Comment (songole) [inline insurance_datasources/README.md]: same as above?

File: insurance_datasources/README.md (hunk 1 above)
Comment (riteshkarval) [reply insurance_datasources/README.md]: Repo name I changed in all the readme's to monitoring-examples to run all examples in a single IDE, but yes I need to change `workspace/monitoring-examples/insurance` to `workspace/monitoring-examples/insurance_datasources`

File: titanic_datasources/README.md [hunk 3]
```diff
@@ -127,30 +127,30 @@
 
 
 ## Section 3: Data Generation
-1. Open [data_generation.ipynb](https://github.com/oneconvergence/dkube-examples/tree/monitoring/titanic/data_generation.ipynb) notebook for generating predict and groundtruth datasets.
+1. Open [data_generation.ipynb](https://github.com/oneconvergence/dkube-examples/tree/monitoring/titanic_datasources/data_generation.ipynb) notebook for generating predict and groundtruth datasets.
 2. In 1st cell, Update Frequency according to what you set in Modelmonitor. If the **d3qatest** tag was provided when creating a modelmonitor, specify the frequency in minutes. For eg: for 5 minutes, specify it as `5m`. If **d3qatest** tag was not specified on the modelmonitor, specify the frequency in hours. use `5h` for 5 hours interval.
```
Comment (songole) [inline titanic_datasources/README.md]: this is incorrect right? there is no d3qatest now. Also, frequency is in the 1st cell of resources.ipynb and not here

File: titanic_datasources/README.md [hunk 4]
```diff
@@ -127,30 +127,30 @@
...
 3. Then Run All Cells. It will start Pushing the data. It uses the data definitions specified in resources.ipynb file.
 
 ## Section 4: Modelmonitoring
 DKube provides Python SDK for creating a modelmonitor programmatically. You could also choose to create a modelmonitor from the DKube UI. Follow one of the following workflows as per your need.
 
 - **Create Modelmonitor using SDK**
-1. From **workspace/titanic/titanic** open [modelmonitor.ipynb](https://github.com/oneconvergence/dkube-examples/tree/monitoring/titanic/modelmonitor.ipynb) and run all the cells. New model monitor will be created.
+1. From **workspace/monitoring-examples/titanic** open [modelmonitor.ipynb](https://github.com/oneconvergence/dkube-examples/tree/monitoring/titanic_datasources/modelmonitor.ipynb) and run all the cells. New model monitor will be created.
```
Comment (songole) [inline titanic_datasources/README.md]: you mean titanic_datasources?

File: titanic_datasources/README.md (hunk 3 above)
Comment (riteshkarval) [reply titanic_datasources/README.md]: yes, fixing

File: titanic_datasources/README.md (hunk 4 above)
Comment (riteshkarval) [reply titanic_datasources/README.md]: monitoring-examples is the repo name. titanic need to be titanic_datasources

**Pull request merged**